### PR TITLE
Add note about map for mobile

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

**Comment by bramkragten** [comment]
What is the reason we don't show in landscape? Is it not useable?

File: site/src/analytics-element.ts [hunk 1]
```diff
@@ -89,6 +89,12 @@ export class AnalyticsElement extends LitElement {
             >
             </analytics-integrations>`
           : ""}
+        ${this._isMobile && this._currentPage === "installations"
+          ? html`<span>
+              The map is only aviable on larger screens, try rotating your
+              device to show it.
+            </span>`
+          : ""}
```
Comment (bramkragten) [inline site/src/analytics-element.ts:97]: Shouldn't this be in `analytics-map`?

Comment (ludeeus) [reply site/src/analytics-element.ts:97]: We don't render in analytics-map?

**Comment by ludeeus** [comment]
As for why.. 
image yeah, fairly unusable?

**Pull request closed without merging**